Code/MinimalLib/docker/Dockerfile

```diff
@@ -21,10 +21,18 @@
 
 ARG RDKIT_GIT_URL="https://github.com/rdkit/rdkit.git"
 ARG RDKIT_BRANCH="master"
+ARG EMSDK_VERSION="latest"
+ARG BOOST_MAJOR_VERSION="1"
+ARG BOOST_MINOR_VERSION="84"
+ARG BOOST_PATCH_VERSION="0"
 
 FROM debian:buster as build-stage
 ARG RDKIT_GIT_URL
 ARG RDKIT_BRANCH
+ARG EMSDK_VERSION
+ARG BOOST_MAJOR_VERSION
+ARG BOOST_MINOR_VERSION
+ARG BOOST_PATCH_VERSION
 
 LABEL maintainer="Greg Landrum <greg.landrum@t5informatics.com>"
```

Comment:
not for this PR, but this should probably be changed at some point. I'm not really the maintainer any more